From 828b425b2e904c92cc2cc866049caa3ec4ec0d8d Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Wed, 24 Apr 2024 09:25:51 +0200
Subject: [PATCH 598/954] refactor(st-clock): remove useless LSEDRV_MEDIUM_HIGH
 define for STM32MP13

Remove LSEDRV_MEDIUM_HIGH definition in clk-stm32mp13.c
It's already defined in include/dt-bindings/clock/stm32mp13-clksrc.h

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 88fc4b6ccd9696cb4974064917df2eeaba986779 refactor(st-clock): remove useless LSEDRV_MEDIUM_HIGH define for STM32MP13]
Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Change-Id: Ie3fa4711930f922fa0733ba7c76d72ec9639e9a5
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/376132
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Tested-by: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 drivers/st/clk/clk-stm32mp13.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/st/clk/clk-stm32mp13.c b/drivers/st/clk/clk-stm32mp13.c
index daf6a3802d..09c53c19ec 100644
--- a/drivers/st/clk/clk-stm32mp13.c
+++ b/drivers/st/clk/clk-stm32mp13.c
@@ -2105,8 +2105,6 @@ static int stm32mp1_init_clock_tree(void)
 	return 0;
 }
 
-#define LSEDRV_MEDIUM_HIGH 2
-
 static int clk_stm32_parse_oscillator_fdt(void *fdt, int node, const char *name,
 					  struct stm32_osci_dt_cfg *osci)
 {
-- 
2.53.0

